src/components/TextEditor.vue

```diff
@@ -178,6 +259,25 @@ export default {
 		 */
 		onEditorReady(editor) {
 			logger.debug('TextEditor is ready', { editor })
+
+			editor.commands.get('mention')?.on('execute', (event, data) => {
+				event.stop()
+				const eventData = data[0]
+				const item = eventData.mention
+				if (eventData.marker === ':') {
+					emojiAddRecent(item)
+					this.editorInstance.execute('InsertItem', item.native, ':')
+				}
+				if (eventData.marker === '/') {
+					getLinkWithPicker(item.id)
+						.then((link) => {
+							this.editorInstance.execute('InsertItem', link, '/')
```

Comment:
```suggestion
							this.editorInstance.execute('InsertItem', link, '/')
							this.editorInstance.editing.view.focus()
```

Seems to "workaround" the reset cursor issue. 

If the reference picker is open window._nc_focus_trap shows three focus traps.
When closing the picker the focus goes back to ckeditor.
I'm still lost what is causing ckeditor to reset the cursor then, especially because we had a selection before.

Reply:
The workaround works 👍